docs/source/en/pipeline_tutorial.mdx

```diff
@@ -247,7 +247,7 @@ For example, if you use this [invoice image](https://huggingface.co/spaces/impir
 ...     image="https://huggingface.co/spaces/impira/docquery/resolve/2359223c1837a7587402bda0f2643382a6eefeab/invoice.png",
 ...     question="What is the invoice number?",
 ... )
-[{'score': 0.42514941096305847, 'answer': 'us-001', 'start': 16, 'end': 16}]
+[{'score': 0.42515, 'answer': 'us-001', 'start': 16, 'end': 16}]
```

Comment:
always better not to use something like `0.42514941096305847` - this is going to break easily